test/functional/3rd-party-repo-add/3rd-party-repo-add-negative.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+@test "TRA003: Negative test, repo add usage" {
+
+	run sudo sh -c "$SWUPD 3rd-party add $SWUPD_OPTS"
+	assert_status_is "$SWUPD_INVALID_OPTION"
+	expected_output=$(cat <<-EOM
+			Error: The positional args: repo-name and URL are missing
+			*.*
+	EOM
+	)
+	assert_regex_in_output "$expected_output"
```

Comment:
why do you need to use assert_regex_in_output here and not just assert_in_output without using the *.*?

Reply:
i guess i can use assert_in_output as well. Thanks will change in next push